src/ngraph/op/fused/conv_fused.cpp

```diff
@@ -92,38 +84,92 @@ op::ConvolutionBias::ConvolutionBias(const shared_ptr<Node>& data_batch,
     , m_with_relu(with_relu)
 {
     constructor_validate_and_infer_types();
+}
+
+op::ConvolutionBias::ConvolutionBias(const shared_ptr<op::Convolution>& conv,
+                                     const shared_ptr<Node>& bias,
+                                     const bool with_relu)
+    : ConvolutionBias(conv->get_argument(0),
+                      conv->get_argument(1),
+                      bias,
+                      conv->get_window_movement_strides(),
+                      conv->get_window_dilation_strides(),
+                      conv->get_padding_below(),
+                      conv->get_padding_above(),
+                      conv->get_data_dilation_strides(),
+                      with_relu)
+{
+}
+
+op::ConvolutionBias::ConvolutionBias(const shared_ptr<Node>& data_batch,
+                                     const shared_ptr<Node>& filters,
+                                     const shared_ptr<Node>& bias)
+    : ConvolutionBias(data_batch,
+                      filters,
+                      bias,
+                      Strides(),
+                      Strides(),
+                      CoordinateDiff(),
+                      CoordinateDiff(),
+                      Strides())
+{
+}
+
+/// Overrides the default shape inference utility provided by FusedOp
+/// based on FusedOp decomposition.
+///
+/// This implementation handles partial shapes and adjust conv attributes
+/// to support simplified ConvolutionBias op construction
+void op::ConvolutionBias::validate_and_infer_types()
```

Comment:
Is this override going to allow/disallow things that are not allowed/disallowed by the default implementation?

Reply:
Does a few more things compared to default. Primarily, setting strides, padding if they are not already specified.